### PR TITLE
Added dashboard wireframe to assets folder 

### LINKED ISSUE #268: Add wireframe to assets folder
As a developer team, when we create the dashboard we want to have a stencil outline wirreframe of how we plan to lay this out so that we can maintain consistency with the original specification.

### PR DESCRIPTION
**Pull request by olliejth**
Added a simple dashboard wireframe to the assets folder.

Updated the README to include a link to the.

Closes #268

### CODE REVIEW

File: assets/starwatch_dashboard_wireframe.png
Comment (CalebMathers) [inline assets/starwatch_dashboard_wireframe.png:1]: Where has this been all my life?

File: dashboard/README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ This folder contains the scripts to run and dockerise the dashboard for the Star
 
 ## Design
 ![ERD](../assets/starwatch_architecture_diagram.png)
-__Figure 1__ - _Starwatch Dashboard Wireframe_: Shows the design for the StarWatch dashboard.
+![Wireframe](../assets/starwatch_dashboard_wireframe.png) Shows the design for the StarWatch dashboard.
```
Comment (CalebMathers) [inline dashboard/README.md:6]: Exquisite link to the image!

**Review by CalebMathers — APPROVED** [review]

File: dashboard/README.md (hunk 1 above)
Comment (AlexOBarnes) [reply dashboard/README.md:6]: Thanks for this update

File: assets/starwatch_dashboard_wireframe.png
Comment (AlexOBarnes) [reply assets/starwatch_dashboard_wireframe.png:1]: This looks really good

**Review by AlexOBarnes — APPROVED** [review]

**Pull request merged**